### PR TITLE
(#487) Add release notes for C4B Azure

### LINKED ISSUE #487: Add release notes for the Chocolatey for Business Azure Environment
## What New Or Updated Would You Like To See?

Add the release note for the Chocolatey for Business Azure Environment to the docs.

## Why Is It Needed?

Release notes for other products are available via the docs site, and this should be no different.

## Additional Context?

N/A

### PR DESCRIPTION
**Pull request by Windos**
## Description Of Changes

This PR adds release notes for the Chocolatey for Business Azure Environment.

## Motivation and Context

The release notes should be available alongside the other related docs for easy reference.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

~~* [ ] Minor documentation fix (typos etc.).~~
~~* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).~~
* [x] New documentation page added.

## Change Checklist

~~* [ ] Requires a change to menu structure (top or left hand side)~~
~~* [ ] Menu structure has been updated~~

## Related Issue

Fixes #487

### CODE REVIEW

File: input/en-us/c4b-environments/azure/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+* [Chocolatey 1.1.0](https://docs.chocolatey.org/en-us/choco/release-notes#march-30-2022)
+* [Chocolatey Licensed Extension 4.1.1](https://docs.chocolatey.org/en-us/licensed-extension/release-notes#april-11-2022)
+* [Chocolatey Central Management 0.8.0](https://docs.chocolatey.org/en-us/central-management/release-notes#february-28-2022)
+* [Chocolatey GUI 1.0.0](https://docs.chocolatey.org/en-us/chocolatey-gui/release-notes#march-21-2022)
+* [Chocolatey GUI Licensed Extension 1.0.0](https://docs.chocolatey.org/en-us/chocolatey-gui-licensed-extension/release-notes#march-21-2022)
+* [Chocolatey Agent 1.0.0](https://docs.chocolatey.org/en-us/agent/release-notes#march-21-2022)
+
+### Features
+
+* Provide easy mechanism for update Certificate - see #78
```
Comment (pauby) [inline input/en-us/c4b-environments/azure/release-notes.md:25]: Can you add the URL link here?

File: input/en-us/c4b-environments/azure/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+### Features
+
+* Upgrades CCM to 0.6.2 - see [#51](https://github.com/chocolatey/c4b-azure/issues/51)
+
+### Bug Fixes
+
+* Fixes an issue with package internalization - see [#54](https://github.com/chocolatey/c4b-azure/issues/54)
+
+### Docs
```
Comment (pauby) [inline input/en-us/c4b-environments/azure/release-notes.md:121]:
```suggestion
### Documentation
```

File: input/en-us/c4b-environments/azure/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+* Updates CCM to version 0.6.1, and updates dependencies
+* Validates Chocolatey Licence _before_ deploying resources
+* Ensured Chocolatey Agent is installed and checking into the CCM installation
+
+### Bug Fixes
+
+* Uses `Restart-WebAppPool` instead of `iisreset`, which fixes a rare issue that failed deployments complaining of insufficient permissions
+
+### Enhancements
```
Comment (pauby) [inline input/en-us/c4b-environments/azure/release-notes.md:150]:
```suggestion
### Improvements
```

File: input/en-us/c4b-environments/azure/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+The first released version! Available on the [Azure Marketplace](https://portal.azure.com/#create/chocolateysoftwareinc1605695330527.c4b_azure_qdec4b-qde) from 2021-07-29.
+
+### Features
+
+* QDE Deployment via the Azure Marketplace
+* Deploys CCM, Nexus, and Jenkins to a VM
+* Takes an SSL certificate and FQDN and configures the environment appropriately
+
+### Docs
```
Comment (pauby) [inline input/en-us/c4b-environments/azure/release-notes.md:175]:
```suggestion
### Documentation
```

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/c4b-environments/azure/release-notes.md (hunk 1 above)
Comment (Windos) [reply input/en-us/c4b-environments/azure/release-notes.md]: Missed one! Fixed it

**Review by pauby — APPROVED** [review]
@Windos awesome work! Thanks for your contribution!

**Pull request merged**